ui-react/packages/atlasmap/src/Atlasmap/Atlasmap.tsx

```diff
@@ -138,6 +138,17 @@ export const Atlasmap: FunctionComponent<IAtlasmapProps> = ({
       onFieldPreviewChange,
       canStartMapping: () => true, // TODO: check that there is at least one target field unmapped and compatible
       onStartMapping: (field) => handlers.onCreateMapping(field, undefined),
+      onCreateNamespace: (docName: string) =>
+        handlers.onCreateNamespace(docName),
+      onEditNamespace: (
+        docName: string,
+        alias: string,
+        uri: string,
+        locationUri: string,
+        isTarget: boolean,
+      ) => handlers.onEditNamespace(docName, alias, uri, locationUri, isTarget),
+      onDeleteNamespace: (docName: string, alias: string) =>
+        handlers.onDeleteNamespace(docName, alias),
```

Comment:
No need for this closure, you can pass the function directly

Reply:
Not sure what you're saying here. Do you mean that you'd rather I pass all 3 of these functions directly to the NamespaceTableView via the appropriate handlers.XXX functions, meaning to create separate attributes in that class to receive these?